.github/workflows/managed-images.yaml

```
@@ -1804,6 +1834,11 @@ jobs:
             TARGETARCH=${{ matrix.arch }}
             NEMOCLAW_MANAGED_IMAGE_CAPABILITY_UNION=1
             NEMOCLAW_MANAGED_IMAGE_RUNTIME_USER=root
+            ${{ matrix.agent == 'openclaw' && format('NEMOCLAW_MCPORTER_AUDIT_RECEIPT_SHA256={0}', steps.mcporter-audit.outputs.receipt_sha256) || '' }}
+            ${{ matrix.agent == 'openclaw' && format('NEMOCLAW_MCPORTER_AUDIT_RAW_REPORT_SHA256={0}', steps.mcporter-audit.outputs.raw_sha256) || '' }}
+          secret-files: |
```

actionlint (1.7.12)

[error] input "secret-files" is not defined in action "Publish managed image digest" defined at "./.github/actions/publish-managed-image-digest". available inputs are "build-args", "build-contexts", "builder", "cache-from", "cache-to", "context", "file", "image", "labels", "platform", "provenance", "sbom"

(action)